tests/openwisp2/settings.py

```diff
@@ -144,6 +144,7 @@
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 
 OPENWISP_MONITORING_MANAGEMENT_IP_ONLY = False
+OPENWISP_MONITORING_AUTO_CLEAR_MANAGEMENT_IP = True
```

Comment:
This line should not be needed because it's the default value, or what am I missing?

Reply:
You're right. I should remove this